nanshe_ipython.ipynb

```diff
@@ -713,6 +713,17 @@
         "\n",
         "        imgs_min, imgs_max = [s.result() for s in status]\n",
         "\n",
+        "    shifts = LazyZarrDataset(fn, \"shifts\")\n",
+        "\n",
+        "    fig, axs = plt.subplots(nrows=shifts.shape[1], sharex=True)\n",
```

Comment:
Create several rows of plots (one for each coordinate) and have them share their X axes (frame #).